jormungandr/src/settings/start/config.rs

```diff
@@ -79,6 +79,10 @@ pub struct P2pConfig {
     /// help the different modules of the P2P topology engine to determine the
     /// best possible neighborhood.
     pub topics_of_interest: Option<BTreeMap<Topic, InterestLevel>>,
+
+    /// Whether to allow non-public IP addresses on the network
+    #[serde(default)]
+    pub allow_private_addresses: bool,
```

Comment:
what is the default value? It's not obvious to me

could you add it in the documentation too? I'd like to see the default set to _false_ by default. Is it the case?

Reply:
Yes, it should be false by default, but I don't know how to specify this explicitly for serde-yaml. I'll add the information to the docs.

Reply:
no worries. Just needs it to be documented in the comment message